es-app/src/guis/GuiMetaDataEd.cpp

```diff
@@ -140,7 +148,7 @@ GuiMetaDataEd::GuiMetaDataEd(Window* window, MetaDataList* md, const std::vector
 
 	std::vector< std::shared_ptr<ButtonComponent> > buttons;
 
-	if(!scraperParams.system->hasPlatformId(PlatformIds::PLATFORM_IGNORE))
+	if(md->getType() == GAME_METADATA && !scraperParams.system->hasPlatformId(PlatformIds::PLATFORM_IGNORE))
```

Comment:
Allow only scraping on a game.

Reply:
Hm. Why? Did it not work? I'd expect that you could either scrape a folder to force it to scrape all games inside it, or scrape a folder as a game (i.e. multi-disc games in the same folder). Unsure if either of these use cases were supported, though, but can you share the current behavior?

Reply:
The check is needed as currently no folder FileData object is created. By enabling it in [L140 of Gamelist.cpp](https://github.com/RetroPie/EmulationStation/pull/841/files/5f6acec5562260ddd1e895388a724b6f17b65c2a#r1431488778) with this PR it is needed to filter the folder FileData out here at this line again.
 
Current behavior:
Currently it is not possible to scrape folders resp. folder-subtrees: <folder> elements are filtered out when present in a gamelist file (because of the L140 see above). 

Scraping folder data is IMHO not really useful as it in the majority of cases will be not matching any ROM name. For scraping a folder with ROMs in one go an external scraper is more suitable.